jdbc/src/test/java/org/apache/iotdb/jdbc/IoTDBDatabaseMetadataTest.java

```diff
@@ -240,7 +240,41 @@ public void device() throws Exception {
     String standard = "column,\n" + "root.vehicle.d0,\n";
     try {
       ResultSet resultSet = databaseMetaData
-          .getColumns(Constant.CATALOG_DEVICE, "vehicle", null, null);
+          .getColumns(Constant.CATALOG_COLUMN, "vehicle", null, null);
+      ResultSetMetaData resultSetMetaData = resultSet.getMetaData();
+      int colCount = resultSetMetaData.getColumnCount();
+      StringBuilder resultStr = new StringBuilder();
+      for (int i = 1; i < colCount + 1; i++) {
+        resultStr.append(resultSetMetaData.getColumnName(i)).append(",");
+      }
+      resultStr.append("\n");
+      while (resultSet.next()) {
+        for (int i = 1; i <= colCount; i++) {
+          resultStr.append(resultSet.getString(i)).append(",");
+        }
+        resultStr.append("\n");
+      }
+      Assert.assertEquals(resultStr.toString(), standard);
+    } catch (SQLException e) {
+      System.out.println(e);
```

Comment:
Simply  print out the exception may bury the error and the TestCase will be regarded as `PASSED`. It's better to replace it with `e.printStackTrace(); fail(e.getMessage());`.